TestingLowerBounds/Hellinger.lean

```diff
@@ -100,6 +99,30 @@ lemma integrable_rpow_rnDeriv_iff [SigmaFinite ν] [SigmaFinite μ] (hμν : μ
     linarith
   · rw [rpow_add (ENNReal.toReal_pos h_zero hx), rpow_one]
 
+lemma integral_rpow_rnDeriv_eq_zero_iff_mutuallySingular [SigmaFinite μ] [SigmaFinite ν]
```

Comment:
```suggestion
-- TODO: generalize to the integral of any nonnegative function of the likelihood ratio
-- which is 0 only at 0.
lemma integral_rpow_rnDeriv_eq_zero_iff_mutuallySingular [SigmaFinite μ] [SigmaFinite ν]
```

Reply:
I directly added the generalized lemma, was this what you had in mind? 
I was not sure what should be the domain of the function. I used `f : ENNReal → ℝ`, in this way it's easier to state the nonnegativity and we don't have to compose with `.toReal`, but on the other hand we need to handle the case where the argument is infinte, so the other hypothesis becomes: `∀ x, f x = 0 ↔ x = 0 ∨ x = ⊤`.

Reply:
Also the name of the new lemma probably needs to be changed.